janusgraph-core/pom.xml

```diff
@@ -81,6 +81,17 @@
             <artifactId>jsr305</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>${caffeine.version}</version>
```

Comment:
(nitpick) You don't need this line because you have already specified the version in the `<dependencyManagement>` section of root `pom.xml`.

Reply:
Right. I.e. this is enough when the version has been already specified in `dependencyManagement`:
```
<dependency>
            <groupId>com.github.ben-manes.caffeine</groupId>
            <artifactId>caffeine</artifactId>
</dependency>
```